src/jaxsim/api/contact.py

```diff
@@ -190,11 +191,11 @@ def in_contact(
 
     links_in_contact = jax.vmap(
         lambda link_index: jnp.where(
-            model.kin_dyn_parameters.contact_parameters.body == link_index,
+            model.physics_model.gc.body == link_index,
```

Comment:
The `functional` branch should no longer access `model.physics_model`.

```suggestion
            model.kin_dyn_parameters.contact_parameters.body == link_index,
```

Reply:
Tests are not failind because this function is not yet part of any test.

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/119/commits/dd0d00b4c76315b48a2a096157ff6531e5e14037